catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/RefreshRegistryEntries.java

```diff
@@ -176,8 +217,8 @@ private boolean hasMatch(Metacard local, List<Metacard> remoteMetacards) {
     }
 
     private RemoteRegistryResults getRemoteRegistryMetacardsMap() throws FederationAdminException {
-        Map<String, Metacard> remoteRegistryMetacards = new HashMap<>();
-        List<String> failedQueries = new ArrayList<>();
+        Map<String, Metacard> remoteRegistryMetacards = new ConcurrentHashMap<>();
```

Comment:
Did we make this and `failedQueries` synchronized because they are returned in the `RemoteRegistryResults` object? As far as I can tell, those are only updated _after_ all the tasks have been run, so they should only be updated by a single thread, and do not seem to be updated outside of this method. If we expect them to be only modified here and not after they are set in `RemoteRegistryResults`, should we make them immutable instead of synchronized?